Source/Shaders/AtmosphereCommon.glsl

```diff
@@ -0,0 +1,165 @@
+uniform vec3 u_radiiAndDynamicAtmosphereColor;
+
+const float G = 0.9;
+const float ATMOSPHERE_THICKNESS = 111e3;
+const vec2 HEIGHT_SCALE = vec2(10e3, 3.2e3); // (Rayleigh, Mie) scales.
+const vec3 BETA_RAYLEIGH = vec3(5.8e-6, 13.5e-6, 33.1e-6); // Better constants from Precomputed Atmospheric Scattering (https://hal.inria.fr/inria-00288758/document)
+const vec3 BETA_MIE = vec3(21e-6);
+const vec3 INTENSITY = vec3(100.0);
+
```

Comment:
It would be nice if all of the magic numbers were moved up here. I'm seeing others in the function and it's hard to know what is tunable vs not.

For example `    float miePhase = 3.0 / (25.1327412287) * ((1.0 - GSq) * (cosAngleSq + 1.0)) / (pow(1.0 + GSq - 2.0 * cosAngle * G, 1.5) * (2.0 + GSq));`